### DISCUSSION TITLE
Customize URL for "oauth-openshift" (login) in the Web Console and its applications

### DISCUSSION THREAD

**Discussion by eduardolucioac** [okd-project/okd#750] [Q&A] [answered]
Hi friends!

The Web Console and its components ("alertmanager", "grafana", "prometheus", etc) always redirect to this URL...

`https://oauth-openshift.apps.mbr.okd.local`

... when login is required.

**NOTE:** The "okd.local" is the cluster's base domain (installation).

Is it possible to configure them to calling another URL? Like this URL below?

`https://oauth-openshift.apps.mbr.domain.my`

**NOTE:** I adjusted the Web Console and its applications configurations and now they work perfectly (including login process) in the ".apps.mbr.domain.my" domain. I just need that when login is needed the applications redirect to `https://oauth-openshift.apps.mbr.domain.my` .

**Thanks! =D**

**Comment by vrutkovs** [comment, marked as answer]
Dupe of #738 - changing basedomain is not supported
